pkg/pillar/go.mod

```diff
@@ -144,3 +144,4 @@ require (
 )
 
 replace github.com/lf-edge/eve/api/go => ../../api/go
+replace github.com/lf-edge/eve/libs => ../../libs
```

Comment:
No, let's not re-introduce this replace.
@deitch removed it so that the SBoM would not be broken (golang gets confused).
Instead we should 1) put the api and libs in separate repos 2) remove the api replace as well.

Reply:
Sorry, was a rebase error on my side.